controllers/mirrorpeer_controller.go

```diff
@@ -179,6 +183,12 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, err
 	}
 
+	err = r.createRoleBindingsForSpoke(ctx, mirrorPeer)
```

Comment:
also need to handle rbac removal when MirrorPeer is deleted.

Reply:
as discussed, will add it in another PR for a cleaner cherry-pick